raiden/transfer/mediated_transfer/mediation_fee.py

```diff
@@ -126,7 +126,7 @@ def _mediation_fee_func(
     assert (
         amount_with_fees is None or amount_without_fees is None
     ), "Must be called with either amount_with_fees or amount_without_fees as None"
-    if balance_out == 0:
+    if balance_out == 0 or receivable == 0:
```

Comment:
A small comment would be nice

Reply:
Is this reasonable, or should I add that we avoid edge cases in the calculation that way? I have no problems to elaborate, I just don't want to bloat the source with unnecessarily detailed comments.